src/manifest/parsers/index.ts

```diff
@@ -114,11 +114,11 @@ export class ManifestParser {
   }
 
   private readAndParseManifest(projectRoot: Uri, targetPath: string) {
-    const manifestLocation = path.join(
-      projectRoot.fsPath,
-      targetPath,
-      DBTProject.MANIFEST_FILE,
-    );
+    let manifestLocation = path.join(targetPath, DBTProject.MANIFEST_FILE);
+    if (!path.isAbsolute(targetPath)) {
+      manifestLocation = path.join(projectRoot.fsPath, manifestLocation);
+    }
+
```

Comment:
Please rewrite back to a const expression and destructure an array onto `path.join`.

Something like (untested):


```suggestion
    const pathParts = [targetpath];
    if (!path.isAbsolute(targetPath)) {
      pathParts.unshift(projectRoot.fsPath);
    }
    const manifestLocation = path.join(...pathParts, DBTProject.MANIFEST_FILE);

```